### PR TITLE
lecture 20240429

### PR DESCRIPTION
**Pull request by cgohla**

### CODE REVIEW

File: lecture20240429.tex [hunk 1]
```diff
@@ -0,0 +1,58 @@
+\chapter{Lecture 20240429}
```
Comment (cgohla) [inline lecture20240429.tex:1]: I don't know what lecture number this is, I just know the date. Maybe we can resolve this later.

**Review by ruimp — APPROVED** [review]
Thanks for the contribution! I believe this is lecture 5, but I'll double check, don't worry about it.

**Comment by cgohla** [comment]
> Thanks for the contribution! I believe this is lecture 5, but I'll double check, don't worry about it.

Oh, I wasn't finished. But I guess I can just send another PR.

**Comment by ruimp** [comment]
I'm sorry, I misunderstood. Thanks again, I won't close the other PR.

**Pull request merged**